recipes/dataeval/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "dataeval" %}
+{% set version = "0.61.0" %}
+{% set sha256 = "bb79f19913a3e6f4732cbba08f9d71006c11cf917081b6b81649a65ee5c39e4f" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: bb79f19913a3e6f4732cbba08f9d71006c11cf917081b6b81649a65ee5c39e4f
```

Reply:
I'm glad grayskull works now!  When we had a more complicated dependency list (due to python 3.8 support) grayskull would raise exceptions trying to resolve the dependency tree.